modules/shared/ChatReplay/ChatReplayPanel.tsx

```diff
@@ -35,7 +36,11 @@ const ChatReplayPanel = (props: ChatReplayPanelProps) => {
         transformResponse: (res) => res,
       });
 
-      setReplayData(parseChatReplay(data.data));
+      const info = await axios.get(props.info, {
```

Comment:
Please make this optional, either by checking if `props.info` is empty/null/undefined, or wrapping the call in a try/catch. As this component is also used in the normal video player, it will crash when trying to load the chat without `props.info` provided.

![image](https://github.com/user-attachments/assets/c584a546-5b3d-4e3f-9594-64dc6850212f)